templates/lune/partials/components/product-title.snip.html

```diff
@@ -0,0 +1,8 @@
+{{#product-title}}
+  <h1 class="lune-product-title" aria-label="{{full_title}}">
+    <span class="relative block caps m0 px3 md-p0 z1 lune-anim-move-up {{#parts.1}}lune-title-has-mobile-hyphen{{/parts.1}}">{{parts.0}}</span>
+    {{#parts.1}}
+      <span class="block caps m0 relative px3 md-p0 z4 lune-anim-move-up">{{.}}</span>
+    {{/parts.1}}
+  </h1>
+{{/product-title}}
```

Comment:
Moved the product title to a partial, as this is being included in more than one place now.